### PR TITLE
Update scarb docs

### LINKED ISSUE #1512: Update dependencies guide to use registries
Update dependencies guide to refer to registries as the default way of adding dependencies to the registry.

https://github.com/software-mansion/scarb/blob/main/website/docs/guides/dependencies.md

Let's consider making this a separate guide, to preserve current guide detailing all nuances of Git-based dependency management  as is.

### PR DESCRIPTION
**Pull request by maciektr**
fixes  #1512

### CODE REVIEW

File: website/docs/guides/dependencies.md [hunk 1]
```diff
@@ -6,56 +6,62 @@ To add a dependency, simply declare it in your `Scarb.toml`.
...
 ```toml
 [dependencies]
-alexandria_math = { git = "https://github.com/keep-starknet-strange/alexandria.git" }
+openzeppelin_merkle_tree = "0.17.0"
 ```
 
-You can pin a Git dependency to concrete commit, branch or a tag using one of the following extra fields that can be
-passed along `git`: `branch`, `tag` and `rev`.
+Using `"0.17.0"` as version requirement means, that you want to use a version `0.17.0` or newer, but not `0.18.0` or
+newer. To accept only a specific version, you can use `"=0.17.0"`. You can learn more about specifying version
```
Comment (THenry14) [inline website/docs/guides/dependencies.md:30]: nitpick: I think phrasing here could be better. Maybe something like "... that you want to use a version `0.17.0` or newer, up until `0.18.0`" or something similar?

File: website/docs/guides/dependencies.md [hunk 2]
```diff
@@ -6,56 +6,62 @@ To add a dependency, simply declare it in your `Scarb.toml`.
...
-If your `Scarb.toml` doesn't already have a `[dependencies]` section, add it, then list the package name and the URL to
-its Git repository.
-This example adds a dependency on the [`alexandria`](https://github.com/keep-starknet-strange/alexandria) package (note
-that Alexandria is a collection of multiple packages, and we will use `alexandria_math` as an example in this guide):
+If your `Scarb.toml` doesn't already have a `[dependencies]` section, add it, then list the package name and the version
+required. You can search for packages to use through the [scarbs.xyz registry website](https://scarbs.xyz/).
+This example adds a dependency on the [`openzeppelin_merkle_tree`](https://github.com/OpenZeppelin/cairo-contracts)
+package (note that OpenZeppelin is a collection of multiple packages, and we will use only one of them as an example in
+this guide). To see all available versions of some package, you can see
+its [registry page](https://scarbs.xyz/packages/openzeppelin_merkle_tree). At the time of writing this guide, the latest
```
Comment (THenry14) [inline website/docs/guides/dependencies.md:21]: Should we also mention to go to "versions" pane, or is it so obvious it's redundant to do so in your opinion?

**Review by THenry14 — APPROVED** [review]
both comments are rather nitpicks, so I approve :+1:

File: website/docs/guides/dependencies.md (hunk 1 above)
Comment (maciektr) [reply website/docs/guides/dependencies.md:30]: Yes, thanks!

**Pull request merged**